src/Parser/Attributes.php

```diff
@@ -73,7 +73,11 @@ public function parseValues(array $data)
             $values[$value['id']] = $this->parseValueName($value['valueNames']);
         }
 
-        $this->results[$attributeId]['values'] = $values;
+        if (!isset($this->results[$attributeId]['values'])) {
```

Comment:
I'm not sure if this has any effect. `$this->results[$attributeId]['values']` is always initialized with an empty array anyway.